plugins/woocommerce/src/Internal/Admin/WcPaySubscriptionsPage.php

```diff
@@ -74,6 +71,56 @@ public function register_subscriptions_page() {
 		}
 	}
 
+	/**
+	 * Returns true if the store is eligible for the WooCommerce subsciptions empty state experiment.
+	 *
+	 * @return bool
+	 */
+	private function is_store_experiment_eligible() {
+		// Ineligible if WooCommerce Payments OR an existing subscriptions plugin is active.
+		$active_plugins         = PluginsHelper::get_active_plugin_slugs();
```

Comment:
Need to confirm whether we are checking for **active** or **installed** plugins.

If **installed** is preferred, can replace this line with https://github.com/woocommerce/woocommerce/blob/4a6c3300aa7a8e824912ae634f5ffbdb6a104082/plugins/woocommerce/src/Admin/PluginsHelper.php#L76

Reply:
The PRD clearly stated that we should check if the plugins are **installed**.

I'll update this and the test instructions to check if plugins are **installed** rather than **active**.

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/32867/commits/e36c5d3b8f848c0b36bfac92fbf1cf6bf037f40b